res/css/views/settings/_DevicesPanel.pcss

```diff
@@ -57,6 +57,12 @@ limitations under the License.
     margin-block: 10px;
     min-height: 35px;
     padding: 0 $spacing-8;
+
+    .mx_DeviceType {
+        /* hide the new device type in legacy device list
```

Comment:
The `DevicesPanel` will be retired after the new device manager reaches feature parity, so allow this hack